win32ss/user/ntuser/menu.c

```diff
@@ -4191,10 +4200,21 @@ static INT FASTCALL MENU_TrackMenu(PMENU pmenu, UINT wFlags, INT x, INT y,
                     {
                         executedMenuId = MENU_ButtonUp( &mt, pmMouse, wFlags);
```

Comment:
Should this call be always done, or only in the case where there is no menu exit (i.e. the `else` case of the if-block below) ?

Reply:
Moving this statement from here into the ```else``` case of the if-block below works as well, so I believe that it would be better to move it.

Reply:
Moved. Thanks.